example/src/main/java/com/klarna/example/MainActivity.java

```diff
@@ -29,6 +30,7 @@
     public static final int REGISTRATION_REQUEST_CODE = 1;
     public static final int PREFERENCES_REQUEST_CODE = 2;
     private static final String USER_TOKEN_KEY = "userToken";
+    private static final String LOG_TAG = MainActivity.class.getName();
```

Comment:
Just For Future knowledge - if you run Proguard and don't keep classes names you can end up with  a TAG like : "aaa"
i know it's not the case but keep that in mind for future :)